src/libImaging/ImPlatform.h

```diff
@@ -61,17 +83,22 @@
 #if SIZEOF_LONG == 8
 #define INT64 long
 #elif SIZEOF_LONG_LONG == 8
-#define INT64 long
+#define INT64 long long
+#else
+#warning Cannot find required 64-bit integer type
```

Comment:
> Add compile warning (not an error) if INT64 wasn't defined.

So, why is this a warning, and not an error like the 16-bit and 32-bit counterparts?

Reply:
Because we don't actually use INT64 anywhere, so it won't cause any problems if someone happens to be using a system that doesn't support it.

Reply:
It was already defined; I didn't add it.

Reply:
Ok, I've created #7180 to remove it.

Reply:
#7180 has been merged. I've created https://github.com/Yay295/Pillow/pull/7 to resolve the conflict.